WORKSPACE

```diff
@@ -217,7 +217,6 @@ maven_install(
     ],
     fetch_sources = True,
     repositories = [
-        "https://jcenter.bintray.com",
```

Comment:
jcenter was shut down awhile ago, and all the files return a 403 now.  I removed this because it was causing bazel to hang resolving the 3rdparty deps.